client/components/mma/accountoverview/updateAmount/SupporterPlusUpdateAmountForm.tsx

```diff
@@ -355,17 +364,21 @@ export const SupporterPlusUpdateAmountForm = (
 								size="medium"
 							/>
 							<p>
-								If you would like to lower your{' '}
-								{monthlyOrAnnual.toLowerCase()} amount below{' '}
-								{minPriceDisplay} please call us via the{' '}
+								If you would like to{' '}
+								{currentAmountIsBelowNewMin
```

Comment:
Would `change` work in both cases here? Only ask because it would simplify the code a bit!

Reply:
yeah possibly .. I took a steer from the stakeholders re the copy here:
<img width="590" alt="Screenshot 2024-07-03 at 11 35 20" src="https://github.com/guardian/manage-frontend/assets/2510683/e5990de1-4284-455b-98a5-f0a4bbedf504">